docs/newbs_getting_started.md

```diff
@@ -104,19 +104,13 @@ You can also try the `qmk-git` package from AUR:
 
 ###  ** FreeBSD **
 
-#### Prerequisites
-
-You will need to install Git and Python. It's possible that you already have both, but if not, run the following commands to install them:
-
-    pkg install git python3
-
-Make sure that `$HOME/.local/bin` is added to your `$PATH` so that locally installed Python packages are available.
-
 #### Installation
 
-Install the QMK CLI by running:
+Install the official FreeBSD package for QMK CLI by running:
```

Comment:
I'm not sure about this wording; as far as I'm aware we (QMK) do not maintain this package, so it cannot really be called "official".

Reply:
Hmm, that's a fair point. The reason why I called it official is that it is maintained by the FreeBSD team and in FreeBSD the ports tree is *the* repository for third-party software. I agree that "official" does not make much sense over here. Before I do some wordsmithing here, is there anything else that's potentially troubling?

Reply:
Otherwise looks good to me.

Reply:
The _official_ word is gone. Thanks for the review!